include/swift/SIL/SILValue.h

```diff
@@ -663,8 +671,10 @@ enum class OperandOwnership {
   /// scope, without ending the outer borrow scope, following stack discipline.
   /// (begin_borrow, begin_apply with @guaranteed).
   NestedBorrow,
-  /// Interior Pointer. Propagates an address into the guaranteed value within
-  /// the base's borrow scope.  (ref_element_addr, open_existential_box)
+  /// Interior Pointer. Propagates an address, pointer, or other trivial value
```

Comment:
I like this comment! One concern though: the way this is written implies that pointer is not trivial. Can you eliminate that implication. My thoughts:

"Propagates an address or a trivial value (e.x.: pointer, no escape closure) that depends *snip*"

Reply:
I'll rewrite this but don't understand the issue because an address is also a trivial

Reply:
```
  /// Interior Pointer. Propagates a trivial value (e.g. address, pointer, or
  /// no-escape closure) that depends on the guaranteed value within the base's
```